docs/specifications/tests/DNSSEC-TP/dnssec11.md

```diff
@@ -37,9 +37,9 @@ It is assumed that *Child Zone* is tested and reported by other test cases:
   not giving a correct DNS response for an authoritative name server, covered
   by [Basic04].
 * This test case will ignore any irregularities in fetching the DS record from
-  parent zoen, covered by [DNSSEC02].
-* This test case will ignore if the DNSKEY and SOA RRsets are signed, covered
-  by [DNSSEC08] and [DNSSEC09].
+  parent zone, covered by [DNSSEC02].
+* This test case will ignore if the DNSKEY and SOA RRsets are unsigned, covered
```

Comment:
The new phrasing with "unsigned" seems odd. It looks like it is implying that DNSSEC08 and DNSSEC09 are covering the test cases if "the DNSKEY and SOA RRsets are unsigned". Even though this is true, I would say that their primary goal is to test if the RRsets are properly signed.
Would you mind revert to "signed". Or do I miss understand the new phrasing?

Reply:
I have done an update.